### PR TITLE
Trained fastscnn model with better performance

### PR DESCRIPTION
**Pull request by ytian8**
Hi, I found this [segmentation paper](https://arxiv.org/abs/2004.14960) from your group very interesting, and implement the FastCNN model according to the paper. Because it is real-time and its performance is strong. I got 72.27 mIoU on Cityscapes validation set, very close to the number you reported in the paper. The model weights can be found at this google drive [location](https://drive.google.com/file/d/1euo43cl6K72camH9wQ55IxXweiZ0U-Hx/view?usp=sharing) if you want to include it in GluonCV.

### CODE REVIEW

**Comment by mli** [comment]
Job PR-1406-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1406/1/index.html 
Code coverage of this PR:  vs. Master:

**Comment by bryanyzhu** [comment]
@ytian8 Thanks a lot for your contribution, it is awesome. I will include the model in our model zoo.
